duckduckgo/duckduckgo-autofill
# Email Protection for Windows

**Reviewer:** @GioSensation 
**Asana:** https://app.asana.com/0/0/1204507131214872/f

## Description
Implements Email Protection for the Windows browser (autofill component).

See also: https://github.com/duckduckgo/windows-browser/pull/658

## Steps to test
1. Checkout https://github.com/duckduckgo/windows-browser/pull/658, make sure `duckduckgo-autofill` submodule is up to date
2. In the menu, click **Email Protection** -> **Enable Email Protection**
3. Login as usual
4. Once logged in, ensure an initial Duck Address is set on the Email web app
5. Logout via the menu or the Email web app
